.ci_support/win_c_compilervs2015cxx_compilervs2015python3.6.yaml

```diff
@@ -18,15 +12,7 @@ libwebp:
 - '0.5'
 numpy:
 - '1.11'
-openblas:
-- 0.2.20
 pin_run_as_build:
-  ffmpeg:
-    max_pin: x.x
-  harfbuzz:
-    max_pin: x.x
-  hdf5:
-    max_pin: x.x.x
```

Comment:
Any ideas why these are all getting dropped?

cc @conda-forge/core

Reply:
hdf5 is only a requirement in meta.yaml for unix, could that be it?

Reply:
similar for openblas, ffmpeg, harfbuzz..

Reply:
Ah good point. Wonder why they were included then. No matter though.